raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -138,14 +139,15 @@ private UnknownAddressSpec() {
         }
     }
 
-    public RaftConfig(AbstractConfig abstractConfig) {
+    public RaftConfig(AbstractConfig abstractConfig, int replicaFetchResponseMaxBytes) {
         this(parseVoterConnections(abstractConfig.getList(QUORUM_VOTERS_CONFIG)),
             abstractConfig.getInt(QUORUM_REQUEST_TIMEOUT_MS_CONFIG),
             abstractConfig.getInt(QUORUM_RETRY_BACKOFF_MS_CONFIG),
             abstractConfig.getInt(QUORUM_ELECTION_TIMEOUT_MS_CONFIG),
             abstractConfig.getInt(QUORUM_ELECTION_BACKOFF_MAX_MS_CONFIG),
             abstractConfig.getInt(QUORUM_FETCH_TIMEOUT_MS_CONFIG),
-            abstractConfig.getInt(QUORUM_LINGER_MS_CONFIG));
+            abstractConfig.getInt(QUORUM_LINGER_MS_CONFIG),
+            replicaFetchResponseMaxBytes);
```

Comment:
Can't we get the `replicaFetchResponseMaxBytes` via passed in `abstractConfig`?

Reply:
done